src/libcommon/utility/utility.cpp

```diff
@@ -640,6 +640,11 @@ QString CommonUtility::getRelativePathFromHome(const QString &dirPath) {
     return QDir::toNativeSeparators(name);
 }
 
+bool CommonUtility::isFileSizeMismatchDetectionEnabled() {
+    static bool enableFileSizeMismatchDetection = !envVarValue("KDRIVE_ENABLE_FILE_SIZE_MISMATCH_DETECTION").empty();
```

Comment:
```suggestion
    static const bool enableFileSizeMismatchDetection = !envVarValue("KDRIVE_ENABLE_FILE_SIZE_MISMATCH_DETECTION").empty();
```

Reply:
Seems like this is not a good practice to const qualify a mathod that return b value: 
![image](https://github.com/user-attachments/assets/b05bd75c-1da0-4aa8-82c4-260c64be3767)

Reply:
Done here: [a8efa66](https://github.com/Infomaniak/desktop-kDrive/pull/269/commits/a8efa66e46deb06285f75acb221030b9560a9bb6)